buildenv/jenkins/copyrightCheck

```diff
@@ -37,35 +37,38 @@ stage('Copyright Check') {
                 returnStdout: true
             ).trim()
             echo FILES
-            def FILES_LIST = FILES.split("\\r?\\n")
-            DATE_YEAR = sh (
-                script: "date +%Y",
-                returnStdout: true
-            ).trim()
-            FILES_LIST.each() {
-                println "Checking file: '${it}'"
-                RESULT = sh (
-                    script: "grep -qE 'Copyright \\(c\\) ([0-9]{4}), ${DATE_YEAR} IBM Corp. and others' '${it}'",
-                    returnStatus: true)
-                if(RESULT != 0) {
-                    echo "FAILURE - Copyright date in file: '${it}' appears to be incorrect"
-                    FAIL = true
-                    BAD_FILES << "${it}"
-                } else {
-                    echo "Copyright date in file: appears to be correct"
+            if (FILES != "") {
```

Comment:
I suggest this would read better if the `if` were inverted; then both the condition is closer to the beginning of both branches. (Same comment for both changes).